docs/sources/update_docs_docker.sh

```diff
@@ -0,0 +1,3 @@
+cd ../../
+docker run --rm -v $PWD:/workspace python:3.7 /bin/bash -c "cd /workspace && pip install -r requirements/requirements_docs.txt && cd docs/sources/ && bash update_docs.sh"
+cd docs/sources
```

Comment:
you don't need this last line. It's bash script ;)

Reply:
thanks, will update in a future pr